integration-tests/environments/react-native/package.json

```
@@ -6,12 +6,12 @@
     "start": "react-native start",
     "lint": "eslint .",
     "test": "npm run test:android && npm run test:ios",
-    "test:ios": "PLATFORM=ios WATCH=false MODE=native npm run common",
-    "test:ios:chrome": "PLATFORM=ios WATCH=false MODE=chrome-debugging npm run common",
-    "test:catalyst": "PLATFORM=catalyst WATCH=false MODE=native npm run common",
-    "test:catalyst:chrome": "PLATFORM=catalyst WATCH=false MODE=chrome-debugging npm run common",
-    "test:android": "PLATFORM=android WATCH=false MODE=native npm run common",
-    "test:android:chrome": "PLATFORM=android WATCH=false MODE=chrome-debugging npm run common",
+    "test:ios": "MOCHA_REMOTE_REPORTER=@realm/mocha-reporter PLATFORM=ios WATCH=false MODE=native npm run common",
+    "test:ios:chrome": "MOCHA_REMOTE_REPORTER=@realm/mocha-reporter PLATFORM=ios WATCH=false MODE=chrome-debugging npm run common",
+    "test:catalyst": "MOCHA_REMOTE_REPORTER=@realm/mocha-reporter PLATFORM=catalyst WATCH=false MODE=native npm run common",
+    "test:catalyst:chrome": "MOCHA_REMOTE_REPORTER=@realm/mocha-reporter PLATFORM=catalyst WATCH=false MODE=chrome-debugging npm run common",
+    "test:android": "MOCHA_REMOTE_REPORTER=@realm/mocha-reporter PLATFORM=android WATCH=false MODE=native npm run common",
+    "test:android:chrome": "MOCHA_REMOTE_REPORTER=@realm/mocha-reporter PLATFORM=android WATCH=false MODE=chrome-debugging npm run common",
```

This is too excessive. It's simpler to just add `--reporter @realm/mocha-reporter` to the `common` NPM script (GitHub UI won't allow me to add that suggestion).

```suggestion
    "test:ios": "PLATFORM=ios WATCH=false MODE=native npm run common",
    "test:ios:chrome": "PLATFORM=ios WATCH=false MODE=chrome-debugging npm run common",
    "test:catalyst": "PLATFORM=catalyst WATCH=false MODE=native npm run common",
    "test:catalyst:chrome": "PLATFORM=catalyst WATCH=false MODE=chrome-debugging npm run common",
    "test:android": "PLATFORM=android WATCH=false MODE=native npm run common",
    "test:android:chrome": "PLATFORM=android WATCH=false MODE=chrome-debugging npm run common",
```